setup.py

```diff
@@ -0,0 +1,39 @@
+from setuptools import setup, find_packages
+
+setup(name='devol',
+    version='0.01',
+    description='Automated deep neural network design via genetic programming.',
+    url='https//github.com/joedav/devol',
+    author='Joe Davidson',
+    author_email='josephddavison@gmail.com',
+    license='MIT',
+
+    classifiers=[
+        # How mature is this project? Common values are
+        #   3 - Alpha
+        #   4 - Beta
+        #   5 - Production/Stable
+        'Development Status :: 3 - Alpha',
+
+        # Indicate who your project is intended for
+        'Intended Audience :: Developers',
+        'Topic :: Scientific/Engineering :: Artificial Intelligence',
+
+        # Pick your license as you wish (should match "license" above)
+         'License :: OSI Approved :: MIT License',
+
+        # Specify the Python versions you support here. In particular, ensure
+        # that you indicate whether you support Python 2, Python 3 or both.
+
+        'Programming Language :: Python :: 3.3',
+        'Programming Language :: Python :: 3.4',
+        'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
+    ],
+    keywords='genetic algorithms',
+
+    install_requires=['keras',
+                      'tensorflow',
```

Comment:
I opted to take this out since tensorflow is not able to just be `pip install`ed. If you disagree, add it back in.